examples/CMakeLists.txt

```diff
@@ -9,4 +9,16 @@ add_custom_target(install_libs
 add_subdirectory(bsoncxx)
 add_subdirectory(mongocxx)
 
-add_custom_target(examples DEPENDS ${MONGOCXX_EXAMPLES} ${BSONCXX_EXAMPLES})
+add_custom_target(examples DEPENDS ${MONGOCXX_EXAMPLE_EXECUTABLES} ${BSONCXX_EXAMPLE_EXECUTABLES})
+
+# Run all mongocxx examples on `make examples`.
```

Comment:
Should we call this target "run-examples" or something? I miss the SCons alias facility when using CMake.